### PR TITLE
Status/2023Q4: Add intro.adoc

### PR DESCRIPTION
**Pull request by lsalvadore**

### CODE REVIEW

**Review by pauamma — APPROVED** [review]
Fixable on commit.

**Comment by lsalvadore** [comment]
What is fixable on commit? I do not see any suggested change anywhere: is something wrong on my side?

File: website/content/en/status/report-2023-10-2023-12/intro.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Here is the fourth 2023 status report, with 18 entries.
+
+This is the last 2023 quarter.
+As you have probably noticed, this status report comes later than usual and with less reports than the preceding quarter.
```
Comment (pauamma) [inline website/content/en/status/report-2023-10-2023-12/intro.adoc:4]: s/less/fewer/

**Comment by pauamma** [comment]
> What is fixable on commit? I do not see any suggested change anywhere: is something wrong on my side?

No, I apparently forgot to submit my nitpick.

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**